src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+  --active_star: url(images/star-active.svg);
+}
```

Comment:
Avoid * selector, actually you don't need url as a variable, but if you want to specify variables use :root selector.
```css
:root {
  --main-color: blue;
}
```